admin/modules/user/users.php

```diff
@@ -2426,6 +2428,13 @@ function toggleBox(action)
 
 				$cache->update_awaitingactivation();
 
+				// send activation mail
+				foreach ($user_mail_data as $mail_data)
```

Comment:
There should be no space between `foreach` and `(` according to our standards. Apart from that the PR is ok